ibis/expr/api.py

```diff
@@ -4032,6 +4032,53 @@ def _table_count(self):
     return ops.Count(self, None).to_expr().name('count')
 
 
+def _table_dropna(self, how='any', subset=None):
+    """
+    Remove rows with null values from the table.
+
+    Parameters
+    ----------
+    how : determine whether a row is removed if there is at least one null
+    value in the row ('any'), or if all row values are null ('all').
+    Options are 'any' or 'all'. Default is 'all'
+    subset : Optional list of columns names to consider.
+    Defaults to all columns.
+
+    Returns
+    -------
+    table : TableExpr
+      New table expression
+    """
+    if how not in {'any', 'all'}:
+        raise ValueError("Please specify how as either 'any' or 'all'.")
+    subset = util.promote_list(subset)
+    invalid = set(subset) - set(self.schema().names)
+    if invalid:
+        raise ValueError(f'Columns {list(invalid)} do not exist in the table.')
+    return ops.DropNa(self, how, subset).to_expr()
+
+
+def _table_fillna(self, value, subset=None):
+    """
+    Fill null values in the table.
+
+    Parameters
+    ----------
+    value : value with which to fill the nulls
+    subset : Optional list of columns to fill. Defaults to all columns.
```

Comment:
Similar to above can you move the parameter description below its name?

Reply:
Fixed